lib/migration_helper.rb

```diff
@@ -0,0 +1,112 @@
+# This file aggregates helper methods which are used in various migrations
+module MigrationHelper
+
+  # Escapes a string literal so that it can be safely fed into PGSQL queries
+  # NOTE: with literals of this type prefix the string delimiters with E, e.g.:
+  # SELECT * FROM plant_populations WHERE plant_population_name = E'#{escaped_name}';
+  # (Because SQL standards are for losers... ;)
+  def escape(string)
```

Comment:
You can score some points now ;)

2015-04-02 21:27 GMT+02:00 Piotr Nowakowski <notifications@github.com>:

> In lib/migration_helper.rb
> <https://github.com/eSpectrum-IT/brassica/pull/107#discussion_r27688839>:
>
> > @@ -0,0 +1,112 @@
> > +# This file aggregates helper methods which are used in various migrations
> > +module MigrationHelper
> > +
> > +  # Escapes a string literal so that it can be safely fed into PGSQL queries
> > +  # NOTE: with literals of this type prefix the string delimiters with E, e.g.:
> > +  # SELECT * FROM plant_populations WHERE plant_population_name = E'#{escaped_name}';
> > +  # (Because SQL standards are for losers... ;)
> > +  def escape(string)
>
> Deprecated in favor of quote_value but thanks all the same - seems no one
> on stackoverflow noticed it... ;)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/eSpectrum-IT/brassica/pull/107/files#r27688839>.
>



-- 
Tomasz Szymczyszyn